### DISCUSSION TITLE
Troubleshooting "UNABLE TO OPEN CONFIG"

### DISCUSSION THREAD

**Discussion by tomkerkhove** [JanDeDobbeleer/oh-my-posh#718] [Troubleshoot] [answered]
What's the best way to troubleshoot "UNABLE TO OPEN CONFIG"?

I'm using same profile on another machine that works but get this error so must be missing something but want to find out what exactly.

**Comment by JanDeDobbeleer** [comment, marked as answer]
I'm going to add the exact context as output this evening. This is most likely a path issue but you'd need to know what exactly 😅

**Reply by tomkerkhove** [reply]
Great, thanks! Happy to test if you want.

**Reply by tomkerkhove** [reply]
Would it be possible to give some line numbers or so?

This certainly helps:
```
Oh My Posh Error:
 ReadMapCB: expect { or n, but found �, error found in #1 byte of ...|��{

  |..., bigger context ...|��{

    "final_space": tru|...
```

But it's still not 100% clear to me.

**Reply by tomkerkhove** [reply]
Probably it's some stupid file encoding :joy:

**Reply by JanDeDobbeleer** [reply]
UTF8 is a requirement. Normally, when exporting in Powershell it exports correctly (although Windows and encoding is still a huge issue in 2021).

**Reply by tomkerkhove** [reply]
I have copied it from GitHub as a raw file so that might be it. I'll check that asap, thanks.

**Reply by tomkerkhove** [reply]
Seems to do the trick, thank you!

**Comment by Karma-Tsering** [comment]
I run below command
Export-PoshTheme -FilePath "~/larserikfinholt.omp.json" -Format json

and after some time, error started poping

ReadMapCB: expect { or n, but found f, error found in #1 byte of ...|flag provid|..., bigger context ...|flag provided but not defined: -config-format
Usag|...

I am still not able to figure out what exactly wrong with me.
your help means a lot to me 
thank you
